packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1617,6 +1689,78 @@ class CheckboxSetting extends StatelessWidget {
   }
 }
 
+/// A widget that represents a check box setting and automatically updates for
+/// value changes to [notifier].
```

Comment:
update dart doc from check box to switch

Reply:
Oops, done!